client/ayon_blender/plugins/load/load_blend_link.py

```diff
@@ -0,0 +1,255 @@
+import bpy
+import os
+import hashlib
+import shutil
+from typing import Dict, List, Optional, Union
+from ayon_core.lib import BoolDef
+from ayon_blender.api import plugin
+
+from ayon_core.pipeline import AVALON_CONTAINER_ID
+from ayon_blender.api.plugin_load import (
+    add_override,
+    load_collection
+)
+from ayon_blender.api.pipeline import (
+    ls,
+    AVALON_CONTAINERS,
+    AVALON_PROPERTY,
+    metadata_update,
+    get_container_name
+)
+
+
+class BlendLinkLoader(plugin.BlenderLoader):
+    """Link assets from a .blend file."""
+
+    product_types = {"*"}
+    representations = {"blend"}
+
+    label = "Link Blend"
+    icon = "code-fork"
+    color = "orange"
+
+    options = [
+        BoolDef(
+            "addOverride",
+            label="Add Override",
+            default=False,
+            tooltip="Add a library override for the loaded asset.",
+        ),
+        BoolDef(
+            "use_arbitrary_loc_for_library",
+            label="Use Arbitrary Location for Library",
+            default=False,
+            tooltip="Store the library in arbitrary location"
+                    "Use with caution when blend file is big.",
+        )
```

Comment:
Sorry, but what's this for? This seems odd.

Reply:
I admit it is odd and extremely hacky. It ensures updating the multiple assets correctly, it may be disturbing but it can temporarily resolve the issue happened in https://github.com/ynput/ayon-blender/pull/105#issuecomment-2996665661 
It may not be pretty solution for sure (we need extra callback for this to check on the linked scene objects if saved for example), but we can still get multiple assets updated without collapsing their loaded libraries when all the linked objects are all same versions (with the same library). Blender would remove unused libraries when they are linked everywhere.

Reply:
I think this is the wrong fix - we don't want to be copying files over.. this won't be trivial to manage, especially in multi-OS environment, etc. I don't think this is a reasonable workaround in any way. @philippe-ynput thoughts?